### PR TITLE
BaSP-MR-32: Tests class get and post

### PR DESCRIPTION
**Pull request by Gianluca27**
Added data to Class seed.
Added tests for Class put and delete endpoints.

### CODE REVIEW

File: src/tests/class.test.js [hunk 1]
```diff
@@ -1 +1,79 @@
...
+const mockIdNotFound = '646a353ecdc4cb3be6be10b3';
+
+const invalidId = 'asdasdasd';
+
+beforeAll(async () => {
+  await Class.collection.insertMany(classSeed);
+});
+
+describe('PUT /api/classes', () => {
+  // test('should return status 400 due to existing data in DB', async () => {
```
Comment (Micaela-Rossi) [inline src/tests/class.test.js:32]: This is the mystic error that can't find?

File: src/seeds/class.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    trainer: '646428fc0b6aa64a90624c05',
+    activity: '6465095739daaa5e7a21051a',
+    slots: 3,
+  },
+  {
+    _id: new mongoose.Types.ObjectId('646a353ecdc4cb3be6be10c3'),
+    day: 'Monday',
+    hour: '17:00',
+    trainer: '646428fc0b6aa64a90624b07',
+    activity: '6465095739daaa5e7a21041a',
```
Comment (nailaghigo) [inline src/seeds/class.js:25]:
```suggestion
    trainer: new mongoose.Types.ObjectId('646428fc0b6aa64a90624a21'),
    activity: new mongoose.Types.ObjectId('6465095739daaa5e7a21031a'),
    slots: 7,
  },
  {
    _id: new mongoose.Types.ObjectId('646a353ecdc4cb3be6be10a7'),
    day: 'Wednesday',
    hour: '17:00',
    trainer: new mongoose.Types.ObjectId('646428fc0b6aa64a90624c05'),
    activity: new mongoose.Types.ObjectId('6465095739daaa5e7a21051a'),
    slots: 3,
  },
  {
    _id: new mongoose.Types.ObjectId('646a353ecdc4cb3be6be10c3'),
    day: 'Monday',
    hour: '17:00',
    trainer: new mongoose.Types.ObjectId('646428fc0b6aa64a90624b07'),
    activity: new mongoose.Types.ObjectId('6465095739daaa5e7a21041a'),
```

File: src/tests/class.test.js [hunk 3]
```diff
@@ -1 +1,79 @@
...
+  //   const response = await request(app).put(`/api/classes/${classSeed[1]._id}`).send(mockClass);
```
Comment (nailaghigo) [inline src/tests/class.test.js:33]: We should make this little change to make this test pass

```suggestion
  //   const response = await request(app).put(`/api/classes/${classSeed[1]._id.toString()}`).send(mockClass);
```

**Review by nailaghigo — CHANGES_REQUESTED** [review]
Some changes to do ...

File: src/seeds/class.js (hunk 2 above)
Comment (Gianluca27) [reply src/seeds/class.js:25]: This was the reason why the test wouldn't pass. Even without adding `.toString()` like it is suggested below. Thanks a lot!

File: src/tests/class.test.js (hunk 3 above)
Comment (Gianluca27) [reply src/tests/class.test.js:33]: I found the change it is not necessary, test will pass without adding it!

**Review by nailaghigo — APPROVED** [review]

**Review by Micaela-Rossi — APPROVED** [review]
Great!

File: src/tests/class.test.js (hunk 1 above)
Comment (Gianluca27) [reply src/tests/class.test.js]: Yess, this is the one! Nai solved it tho =)

**Pull request merged**